### ISSUE TITLE
RESTART Shell right after the beginning of training phase

### ISSUE THREAD

**Issue by MRLoghmani** [fchollet/keras#2597]
Hi,
I recently started to use Keras, so I'm still confused about some of the mechanisms. Probably it is a very stupid error, but still I couldn't find any relevant post for solving my problem. 
As you can read from the title my training phase stops right after starting without any error message.
This is how my output looks like:
![untitled](https://cloud.githubusercontent.com/assets/6833023/15002777/eb27e4ac-11df-11e6-872a-1e0a06df3bf9.png)

This is how I created my network:

    model = Sequential()

    model.add(GRU(hidden_neurons_1, input_dim=in_neurons, return_sequences=True))
    model.add(Dropout(0.2))
    model.add(GRU(hidden_neurons_2, input_dim=hidden_neurons_1, return_sequences=True))
    model.add(Dropout(0.2))
    model.add(TimeDistributedDense(out_neurons, activation = 'softmax'))

    model.compile( loss = 'mse', optimizer = 'rmsprop',  metrics=["accuracy"])

    model.fit(newNPZ, newY,batch_size = 1,nb_epoch=10, verbose = 1)

where newNPZ and newY have dimensions (131,45,225) and (131,45,4) respectively

What could be the source of the problem?
